### DISCUSSION TITLE
I want JSON output from --sentence

### DISCUSSION THREAD

**Discussion by Dean-Corso** [Purfview/whisper-standalone-win#132] [Q&A] [answered]
@Purfview.

so I told you already what the problem is about the JSON output but if you prefer to hear it again in a short statement then here we go.

The JSON output is not written in same style like the sentences in SRT / VTT files. I would like to request whether you could output the JSON file (optional all files) on same way (whole sentences) when using the --sentence command line parameter. Thanks.

**Comment by Purfview** [comment, marked as answer]
You wrote that it's for [Videogrep](https://github.com/antiboredom/videogrep), it supports SRT/VTT files, why you don't use them?

Btw, `--sentence` shouldn't have affect on JSON. So, I edited the title.

**Comment by Dean-Corso** [comment]
Yes that's correct. Alright so now I can see it that it also works with the SRT / VTT files to make cuts with sentences with Videogrep. Ok that's a bit confusing so before it was just working with the JSON file. Possibly my fault that I did not notice it before. What a surprise now. 😃 Sorry about that @Purfview. So it seems that I don't need any adjusted JSON file in this case for Videogrep only when I want to search and output words fragments.

Maybe my request in this case was a little superfluous but maybe not at all and you could / should keep that problem with the JSON file output on your to do list so that you get later all same out if possible.

**Reply by Purfview** [reply]
> Yes that's correct. Alright so now I can see it that it also works with the SRT / VTT files to make cuts with sentences with Videogrep.

Problem solved.
Keep posts short next time. 😝

> ...that problem with the JSON file output...

There is no problem with the JSON file output.
